### PR TITLE
Added SUSHI USDC-USDT on Fuse

### PR DESCRIPTION
**Pull request by shadoweddev**

### CODE REVIEW

File: src/features/configure/vault/fuse_pools.js [hunk 1]
```diff
@@ -33,10 +33,45 @@ export const fusePools = [
...
+    oraclePrice: 0,
+    withdrawalFee: '0%',
+    depositsPaused: false,
+    status: 'active',
+    platform: 'SushiSwap',
+    assets: ['USDC', 'USDT'],
+    risks: [
+      'COMPLEXITY_LOW',
+      'BATTLE_TESTED',
+      'IL_LOW',
```
Comment (roman-monk) [inline src/features/configure/vault/fuse_pools.js:60]: IL_NONE, MC_LARGE ?

**Comment by roman-monk** [comment]
missing convenient routes functions in strat
https://github.com/beefyfinance/beefy-contracts/blob/master/contracts/BIFI/strategies/Common/StrategyCommonChefLP.sol#L294

would be nice to add in future vaults

**Comment by roman-monk** [comment]
outputToNative route is wrong also.

and live vault SUSHI-ETH is using it losing rewards, should be upgraded.

also harvestWithCallFeeRecipient

**Comment by shadoweddev** [comment]
I have outputToNative set to [SUSHI,FUSE] to handle if farms turn on sushi rewards, as suggested by weso. 
With this structure we wont need to update the route and if no sushi are present it should still harvest Fuse

I am upgrading SUSHI-WETH now + I will commit a fix up for the requested changed once I understand the issue w/harvestWithCallFeeRecipient. 
I can't see any glaring issues with this function, would you like me remove it or integrate in testing?

**Comment by MirthFutures** [comment]
There is no Sushi-FUSE liquidity, I meant we should have the correct sushi route to swap to native. I didnt look to see what the sushi pair was. It should be SUSHI-ETH-FUSE.

**Pull request closed without merging**